cost-analyzer/values.yaml

```diff
@@ -579,51 +579,59 @@ kubecostModel:
-          # trace:  # Tracing configuration
-            # enable: true  # Enable tracing
-          # part_size: 134217728  # Part size for multipart uploads
-      # - type: GCS  # Type of storage provider
-        # config:
-          # bucket: GCP_object_store_bucket  # GCS bucket name
-          # service_account:  # GCS service account credentials
-            # type: service_account  # Type of service account
-            # project_id: GCP_object_store_project_id  # Project ID
-            # private_key_id: GCP_object_store_private_key_id  # Private key ID
-            # private_key: GCP_object_store_privateKEY  # Private key
-            # client_email: GCP_object_store_client_email  # Client email
-            # client_id: GCP_object_store_client_id  # Client ID
-            # auth_uri: https://accounts.google.com/o/oauth2/auth  # Auth URI
-            # token_uri: https://oauth2.googleapis.com/token  # Token URI
-            # auth_provider_x509_cert_url: https://www.googleapis.com/oauth2/v1/certs  # Auth provider cert URL
-            # client_x509_cert_url: GCP_object_store_client_x509_cert_url  # Client cert URL
-      # - type: AZURE  # Type of storage provider
-        # config:
-          # storage_account: AZ_object_store_storage_account  # Azure storage account name
-          # storage_account_key: AZ_object_store_storageAccount_key  # Azure storage account key
-          # container: AZ_object_store_container  # Azure container name
-          # max_retries: 0  # Max retries for operations
+  #   # AWS EXAMPLE
+  #   type: S3
+  #   config:
+  #     bucket: "kubecost-federated-storage-bucket"
+  #     endpoint: "s3.amazonaws.com"
+  #     region: "us-east-1"
+  #     access_key: "<your-access-key>"
+  #     secret_key: "<your-secret-key>"
+  #     insecure: false
+  #     signature_version2: false
+  #     put_user_metadata:
+  #         "X-Amz-Acl": "bucket-owner-full-control"
+  #     http_config:
+  #       idle_conn_timeout: 90s
+  #       response_header_timeout: 2m
+  #       insecure_skip_verify: false
+  #     trace:
+  #       enable: true
+  #     part_size: 134217728
+  #   # AZURE EXAMPLE
+  #   type: AZURE
+  #   config:
+  #     storage_account: ""
+  #     storage_account_key: ""
+  #     container: ""
+  #     max_retries: 0
+  #   # GCP EXAMPLE
+  #   type: GCS
+  #   config:
+  #     bucket: "kubecost-federated-storage-bucket"
+  #     service_account: |-
+  #       {
+  #         "type": "service_account",
+  #         "project_id": "...",
+  #         "private_key_id": "...",
+  #         "private_key": "...",
+  #         "client_email": "...",
+  #         "client_id": "...",
+  #         "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  #         "token_uri": "https://oauth2.googleapis.com/token",
+  #         "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  #         "client_x509_cert_url": ""
+  #       }
```

Comment:
The previous example of `federatedStorageConfig` showed a list of configs. However it should not be a list because the user should only be specifying a single config. I updated the comments with examples of how to configure an AWS/Azure/GCP config.